data/json/vehicle_parts.json

```diff
@@ -5334,5 +5334,24 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "min": 1, "max": 2}
         ]
+    },
+    {
+        "type" : "vehicle_part",
+        "id" : "vehicle_scoop",
+        "name" : "scoop",
+        "symbol" : "R",
+        "color" : "light_gray",
+        "broken_symbol" : ";",
+        "broken_color" : "dark_gray",
+        "durability" : 20,
+        "epower" : -5,
```

Comment:
That's an incredibly low epower cost. The scoop doesn't have a computer to make it work only when there are items to pick up, so it should use at least as much as a single electric motor running at at least 50%.